katalog/prometheus-adapter/kustomization.yaml

```diff
@@ -19,6 +21,25 @@ patchesStrategicMerge:
     namespace: monitoring
   spec:
     replicas: 1
+    template:
+      spec:
+        containers:
+          - name: prometheus-adapter
+            resources:
+              limits:
+                memory: 512Mi
```

Comment:
```suggestion
                # we increase the memory limit because we have more rules than the default from upstream and they use slightly more memory in the adapter's container.
                memory: 512Mi
```